ci-scripts/docker-image/setup.sh

```diff
@@ -25,28 +25,22 @@ apt-get install -y \
     libx11-dev \
     pkg-config \
     python \
+    python-mako \
     python-pip \
+    python-setuptools \
+    python-voluptuous \
+    python-yaml \
     software-properties-common
 
-# Build freetype with subpixel rendering enabled
-curl -sSfL -o ft.tar.bz2 \
-    https://download.savannah.gnu.org/releases/freetype/freetype-2.8.1.tar.bz2
-tar xjf ft.tar.bz2
-cd freetype-2.8.1
-# Need to respect 80-char line limit for servo-tidy, or this would be neater
-SUBPIXEL_OPTION="FT_CONFIG_OPTION_SUBPIXEL_RENDERING"
-sed --in-place="" \
-    -e "s/.*${SUBPIXEL_OPTION}.*/#define ${SUBPIXEL_OPTION}/" \
-    include/freetype/config/ftoption.h
-./configure
-make
-make install
-
-# Replace the system libfreetype with the one we just built
-cd /usr/lib/x86_64-linux-gnu/
-rm -f libfreetype.so.6
-ln -s /usr/local/lib/libfreetype.so.6
+# Get freetype 2.8 with subpixel rendering enabled. The SNAPSHOT_ARCHIVE
```

Comment:
Maybe add a comment why 2.8, so that someone else doesn't have to rediscover why next time :)